fhir-server-test/src/test/java/com/ibm/fhir/cli/test/FHIRCliTest.java

```diff
@@ -296,15 +295,15 @@ private void runTest(String methodName, String... args) throws Exception {
             throw t;
         }
     }
-    
+
     private void verifyConsoleOutput(String... expectedMsgs) {
         for (int i = 0; i < expectedMsgs.length; i++) {
             if (!consoleOutput.contains(expectedMsgs[i])) {
                 fail("Console output didn't contain: '" + expectedMsgs[i] + "'");
             }
         }
     }
-    
+
     private void verifyFileContents(String fname, String... expectedMsgs) throws Exception {
         String fileContents = Files.readFile(new FileInputStream(dirPrefix(fname)));
```

Comment:
does readFile execute an AutoCloseable?

Reply:
I don't think so, but because it will always be released after the test codes exit, so I think it should be fine.